### PR TITLE
set a max-height for textarea and overflow-y

### PR DESCRIPTION
**Pull request by beerpratapsingh**

### CODE REVIEW

File: src/less/modules/form-component.less [hunk 1]
```diff
@@ -115,6 +115,8 @@
 			}
 		}
 		textarea {
+			max-height: 130px;
```
Comment (snehilmodani) [inline src/less/modules/form-component.less]: can you try and make this configurable?

Comment (snehilmodani) [reply src/less/modules/form-component.less]: try using a user specified class or property for textarea to make this css effect

**Comment by beerpratapsingh** [comment]
Done. Added class through config. for all elements.

**Comment by beerpratapsingh** [comment]
Done.

**Comment by beerpratapsingh** [comment]
Done.

**Pull request merged**